### PR TITLE
Updated Doc: Change Formatting of Source Paper Link

### PR DESCRIPTION
**Pull request by bradhgq**
Changed direct arxiv link to format with paper title and authors that looks like:

See Source [Batch Normalization: Accelerating Deep Network Training by Reducing 
Internal Covariate Shift, S. Ioffe, C. Szegedy](http://arxiv.org/abs/1502.03167).

### CODE REVIEW

File: tensorflow/python/ops/nn_impl.py [hunk 1]
```diff
@@ -1218,6 +1217,9 @@ def batch_normalization(x,
       `tf.nn.moments(..., keep_dims=False)` during training, or running averages
       thereof during inference.
 
+  See Source [Batch Normalization: Accelerating Deep Network Training by Reducing
```
Comment (kyscg) [inline tensorflow/python/ops/nn_impl.py]:
```suggestion
  See Source: [Batch Normalization: Accelerating Deep Network Training by Reducing 
```

File: tensorflow/python/ops/nn_impl.py [hunk 2]
```diff
@@ -1218,6 +1217,9 @@ def batch_normalization(x,
       `tf.nn.moments(..., keep_dims=False)` during training, or running averages
       thereof during inference.
 
+  See Source [Batch Normalization: Accelerating Deep Network Training by Reducing 
+  Internal Covariate Shift, S. Ioffe, C. Szegedy](http://arxiv.org/abs/1502.03167).
```
Comment (kyscg) [inline tensorflow/python/ops/nn_impl.py]:
```suggestion
  Internal Covariate Shift; S. Ioffe, C. Szegedy](http://arxiv.org/abs/1502.03167).
```

File: tensorflow/python/ops/nn_impl.py [hunk 3]
```diff
@@ -1255,7 +1257,8 @@ def fused_batch_norm(
     name=None):
   r"""Batch normalization.
 
-  As described in http://arxiv.org/abs/1502.03167.
+  See Source [Batch Normalization: Accelerating Deep Network Training by Reducing
```
Comment (kyscg) [inline tensorflow/python/ops/nn_impl.py]:
```suggestion
  See Source: [Batch Normalization: Accelerating Deep Network Training by Reducing 
```

File: tensorflow/python/ops/nn_impl.py [hunk 4]
```diff
@@ -1255,7 +1257,8 @@ def fused_batch_norm(
     name=None):
   r"""Batch normalization.
 
-  As described in http://arxiv.org/abs/1502.03167.
+  See Source [Batch Normalization: Accelerating Deep Network Training by Reducing 
+  Internal Covariate Shift, S. Ioffe, C. Szegedy](http://arxiv.org/abs/1502.03167).
```
Comment (kyscg) [inline tensorflow/python/ops/nn_impl.py]:
```suggestion
  Internal Covariate Shift; S. Ioffe, C. Szegedy](http://arxiv.org/abs/1502.03167).
```

**Review by kyscg — CHANGES_REQUESTED** [review]

**Comment by rthadur** [comment]
@BradHuang1999 please sign CLA

**Comment by BradHuang1999** [comment]
@rthadur Already have. It seems that since I have integrated @kyscg’s change, we will need his consent as well

**Comment by kyscg** [comment]
I've signed it

**Review by kyscg — APPROVED** [review]

**Review by ymodak — APPROVED** [review]

**Comment by kyscg** [comment]
@rthadur @gbaned Gentle ping to pull

**Comment by rthadur** [comment]
@BradHuang1999 can you please resolve conflicts.

**Comment by BradHuang1999** [comment]
Done.

**Comment by kyscg** [comment]
@rthadur It's done, gentle ping to set CLA's to yes and pull. Thankss

**Comment by kyscg** [comment]
@rthadur It's been a while, could you pull this in, all changes have been approved
